public/site.webmanifest

```diff
@@ -48,8 +48,38 @@
             "type": "image/png"
         }
     ],
+    "shortcuts": [
+        {
+            "name": "Brew AeroPress",
```

Comment:
Let's remove the "Brew" from here and make the longer name equal to the shorter name.